wordpress-mobile/WordPress-iOS
# WordPress Authenticator to pasteboard fix branch

Addresses most of #15438

To test:

* Enter username and password for an account with 2FA enabled.
* On the 2FA screen leave the app - You should _not_ see any pop ups about pasteboard usage when leaving the app.
* Copy a URL from Safari.
* Come back to the app - You should _not_ see any alert for pasteboard usage.
* Leave the app and copy a 2FA code.
* Return to the app - You _should_ see an alert for pasteboard usage and the code should be pasted into the field.

PR submission checklist:
